lib/Mojo/WebSocketProxy/Backend/JSONRPC.pm

```diff
@@ -92,6 +92,8 @@ sub call_rpc {
     my $after_got_rpc_response_hook  = delete($req_storage->{after_got_rpc_response})  || [];
     my $before_call_hook             = delete($req_storage->{before_call})             || [];
     my $rpc_failure_cb               = delete($req_storage->{rpc_failure_cb});
+    # This flag will decide whether it should send back response.
```

Comment:
can you expand this comment a bit.  Maybe add when this would be useful. and explain that it stops the response ever being sent to the client.

Reply:
Done.